symphony/bdk/core/service/obo_services.py

```diff
@@ -0,0 +1,69 @@
+from symphony.bdk.core.auth.auth_session import OboAuthSession
+from symphony.bdk.core.client.api_client_factory import ApiClientFactory
+from symphony.bdk.core.config.model.bdk_config import BdkConfig
+from symphony.bdk.core.service.connection.connection_service import OboConnectionService
+from symphony.bdk.core.service.message.message_service import OboMessageService
+from symphony.bdk.core.service.stream.stream_service import OboStreamService
+from symphony.bdk.core.service.user.user_service import OboUserService
+from symphony.bdk.core.service_factory import OboServiceFactory
+
+
+class OboServices:
+    """Entry point for OBO-enabled services, see
+    `the list of OBO-enabled endpoints <https://developers.symphony.com/restapi/reference#obo-enabled-endpoints`_
+    """
+
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
```

Comment:
how do we document that you should use async with here?

Reply:
I added a code samples in `docs/authentication.md`.